templates/impala/base.html

```diff
@@ -56,12 +56,11 @@
         data-collect-timings="{{ url('amo.timing.record') }}:{{ collect_timings_percent }}"
         {% block bodyattrs %}{% endblock %}>
 
-    <div id="tabzilla-wrapper">
+    <div id="main-wrapper">
       <div id="page" class="c">
```

Comment:
Lots of duplicate tabzilla code in these two `base.html` files. Any chance you can factor it out?